test/e2e/util.go

```diff
@@ -576,6 +540,53 @@ func RunRC(c *client.Client, name string, ns, image string, replicas int) error
 	return nil
 }
 
+func ResizeRC(c *client.Client, ns, name string, size uint) error {
+	By(fmt.Sprintf("Resizing replication controller %s to %d", name, size))
+	resizer, err := kubectl.ResizerFor("ReplicationController", kubectl.NewResizerClient(c))
+	if err != nil {
+		return err
+	}
+	waitForReplicas := kubectl.NewRetryParams(5*time.Second, 5*time.Minute)
```

Comment:
Either expose this so I can control the resize to return without waiting, or force everyone to wait and delete waitForRCPodsRunning? Otherwise I'm not sure how I'd use the 2 together